tools/pv_tree_generator/text_format.py

```diff
@@ -0,0 +1,158 @@
+# Lint as: python3
+
+# Copyright 2020 Google LLC
```

Comment:
can you also add the test file in this PR. Can check how the test is setup is done for the browser server: 
https://github.com/datacommonsorg/website/blob/master/quantity_range_test.py

to run the test: https://github.com/datacommonsorg/website/blob/master/run_test.sh#L20-L23

you can add a similar section in this file as https://github.com/datacommonsorg/website/blob/master/Dockerfile#L7-L10

This will enable the test to run when you send a PR

Reply:
I added the test files. I'll add the test for function "filter_and_sort" after I finish the build-tree functions.